services/brig/src/Brig/API/Handler.hs

```diff
@@ -76,7 +76,7 @@ type JSON = Media "application" "json"
 -- TODO: move to libs/wai-utilities?  there is a parseJson' in "Network.Wai.Utilities.Request",
 -- but adjusting its signature to this here would require to move more code out of brig (at least
 -- badRequest and probably all the other errors).
-parseJsonBody :: FromJSON a => Request -> Handler a
+parseJsonBody :: FromJSON a => JsonRequest a -> Handler a
```

Comment:
Nice! I ran into a copy paste error the other day that this would have prevented 👍 MORE TYPES! Yay!